packages/roosterjs-content-model-types/lib/enum/PasteType.ts

```diff
@@ -20,4 +20,9 @@ export type PasteType =
     /**
      * If there is a image uri in the clipboard, paste the content as image element
      */
-    | 'asImage';
+    | 'asImage'
+
+    /**
+     * If there is plain text with links in the clipboard, paste the content as plain text with clickable links
+     */
```

Comment:
Ok I think now I understand what you mean. You want to paste as plain text to remove all formats, but only keep links?

I don't know if it is a good idea. But I feel this should be done in your own code, but not as part of roosterjs.

Reply:
Yes, I thought to help submit a PR to add a new option where links can appear in plain text/clickable. If you don't think this is a good idea to continue, I can close this PR?

I thought when you mentioned in the issue that he could write his own plugin and contribute back- this option could be a good thing to have in roosterjs  :).

Reply:
please feel free to close